variants/twatch/pins_arduino.h

```diff
@@ -0,0 +1,41 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        20
+#define NUM_ANALOG_INPUTS       16
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(esp32_adc2gpio[(p)]):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+// touch screen
+#define TP_SDA              14
+#define TP_SCL              15
+#define TP_INT              38
+
+// Interrupt IO port
+#define RTC_INT             37
+#define APX20X_INT          35
+#define BMA42X_INT1         39
+#define BMA42X_INT2         4
+
+static const uint8_t KEY_BUILTIN = 36;
+//Serial1 Already assigned to GPS LORA
+static const uint8_t TX = 33;
+static const uint8_t RX = 34;
```

Comment:
if this is Serial1, I will suggest that you add:
```cpp
#define TX1 TX
#define RX1 RX
```

So when someone calls `Serial1.begin(baud)` it will run with those pins

Reply:
Ok, thank you for your suggestion, it has been modified.